lib/mad_cart/store/base.rb

```diff
@@ -42,6 +42,41 @@ def init_connection
         @connection ||= execute_delegate(klass.connection_delegate, @init_args)
       end
 
+      def validate_by_path(path)
+        check_for_errors do
+          connection.get(path)
+        end
+        return true
```

Comment:
I was specifically referring to the use of the `return` keyword. Returning the values `true` and `false` is cool, but do we need `return`?